vars/production.yml

```diff
@@ -0,0 +1,5 @@
+---
+subnets:
+  - subnet-bf8c5ce6
+  - subnet-916bfcf4
+  - subnet-82dc63f5
```

Comment:
The `production.yml` and `preview.yml` subnets are identical - is that right? And if so do we need both files, or could we just have one file that is used for both stages?